internal/controllers/addon/package_operator_reconciler.go

```diff
@@ -43,19 +49,52 @@ func (r *PackageOperatorReconciler) Reconcile(ctx context.Context, addon *addons
 }
 
 func (r *PackageOperatorReconciler) makeSureClusterObjectTemplateExists(ctx context.Context, addon *addonsv1alpha1.Addon) error {
+	addonDestNamespace := extractDestinationNamespace(addon)
+
+	if len(addonDestNamespace) < 1 {
+		return errors.New(fmt.Sprintf("no destination namespace configured in addon %s", addon.Name))
```

Comment:
you can just use `return fmt.Errorf("no destination namespace configured in addon %s", addon.Name)`